commands/Moderation/role.js

```diff
@@ -0,0 +1,27 @@
+exports.run = async (client, msg, [user, role]) => {
+  if (!msg.guild.roles.exists("name", role)) return msg.reply("There is no role by that name in this server.");
+  if (user.roles.find("name", role)) {
```

Comment:
Assign this as a constant, and then in the message sent, use it. Instead of fetching the role for third time.